pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -31,6 +31,12 @@ import (
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Namespaced
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=`.status.readyToUse`,description="Indicates if a snapshot is ready to be used to restore a volume."
+// +kubebuilder:printcolumn:name="Source",type=string,JSONPath=`.spec.source.persistentVolumeClaimName`,description="Specifies where a snapshot will be created from."
+// +kubebuilder:printcolumn:name="RestoreSize",type=string,JSONPath=`.status.restoreSize`,description="Represents the complete size of the snapshot in bytes."
+// +kubebuilder:printcolumn:name="SnapshotClass",type=string,JSONPath=`.spec.volumeSnapshotClassName`,description="The name of the VolumeSnapshotClass requested by the VolumeSnapshot."
+// +kubebuilder:printcolumn:name="SnapshotContent",type=string,JSONPath=`.status.boundVolumeSnapshotContentName`,description="The name of the VolumeSnapshotContent to which this VolumeSnapshot is bound."
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
```

Comment:
s/Age/CreationTime

Reply:
This should be the CreationTime from the VolumeSnapshot Status.

Reply:
I've left the column title as `Age`, as this is consistent with the rest of the `get` outputs. The source has been updated to point to `.status.creationTime`.